public/index.pug

```diff
@@ -56,6 +78,10 @@ html(lang="en-US")
+        li(class="header__list-item")
+          button(class="header__list-icon popup-trigger" title="Third-Party Extensions")
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
+              path(d="m 16.513244,23.995886 c 0.486316,0 0.884743,-0.409325 0.884743,-0.907096 v -4.971547 c 0.302726,-2.68015 1.419885,-1.884127 2.734304,-1.055193 3.177652,2.003429 5.288926,-3.266369 2.720633,-4.891327 -2.015573,-1.275283 -2.888597,0.917381 -4.365122,0.691121 -0.570297,-0.08845 -0.966771,-0.719918 -1.091768,-1.680494 V 7.5899898 c 0,-0.4998286 -0.398427,-0.9070963 -0.884743,-0.9070963 H 12.446943 C 9.0915613,6.246829 12.07,4.3441893 12.07,2.5731917 12.071953,1.1518684 10.816126,0 9.267338,0 7.720503,0 6.300618,1.1518684 6.300618,2.5731917 c 0,1.6702091 3.0819514,3.6736373 -0.3203043,4.1097018 H 0.88474257 C 0.39842704,6.6828935 -1.4317642e-7,7.0901612 -1.4317642e-7,7.5899898 V 11.173123 C 0.25975876,12.701405 1.2675452,13.055194 2.5585274,12.047309 c 0.4355356,-0.341447 1.1698916,-0.73843 1.7011278,-0.73843 1.3847297,0 2.5097006,1.285568 2.5097006,2.871443 0,1.585876 -1.1230178,3.221119 -2.5097006,3.221119 -0.4941278,0 -0.9550534,-0.164553 -1.3456683,-0.450464 0,0.121358 -2.42181229,-2.231745 -2.91398704317642,0.647927 v 5.493999 C -1.4317642e-7,23.592732 0.39842704,24 0.88474257,24 3.6131876,24 6.3377264,24 9.0642182,23.997894 9.1716373,23.473382 8.822037,22.872765 8.4939205,22.352368 5.9119561,18.25295 15.956618,17.843626 13.374654,22.479896 c -0.12695,0.228317 -0.2539,0.450463 -0.351554,0.666438 -0.123044,0.269455 -0.205073,0.59856 -0.187495,0.85156 z")
```

Comment:
This path could be simplified with [SVGO](https://jakearchibald.github.io/svgomg/):

```suggestion
              path(d="M16.513 23.996a.9.9 0 0 0 .885-.907v-4.972c.303-2.68 1.42-1.884 2.734-1.055 3.178 2.003 5.29-3.266 2.72-4.891-2.015-1.276-2.888.917-4.364.69-.57-.088-.967-.72-1.092-1.68V7.59c0-.5-.398-.907-.885-.907h-4.064c-3.355-.436-.377-2.339-.377-4.11C12.072 1.152 10.816 0 9.267 0 7.721 0 6.301 1.152 6.301 2.573c0 1.67 3.082 3.674-.32 4.11H.884A.898.898 0 0 0 0 7.59v3.583c.26 1.528 1.268 1.882 2.559.874.435-.341 1.17-.738 1.7-.738 1.385 0 2.51 1.285 2.51 2.871s-1.123 3.221-2.51 3.221c-.493 0-.954-.164-1.345-.45 0 .121-2.422-2.232-2.914.648v5.494c0 .5.398.907.885.907 2.728 0 5.453 0 8.18-.002.107-.525-.243-1.125-.571-1.646-2.582-4.1 7.463-4.508 4.88.128-.126.228-.253.45-.35.666-.124.27-.206.599-.188.852z")
```